src/PresentationalComponents/RulesTable/RulesTable.cy.js

```diff
@@ -0,0 +1,359 @@
+import React from 'react';
```

Comment:
In the future I would like to work with you on compartmentalizing as much as we can. This is the first implementation so this is fine for now, after this is merged, in the next refactor PR I'd like to work on this.